doc/docs/master/how-tos/mount-volume.md

```diff
@@ -0,0 +1,184 @@
+and make it available in your pipeline worker by using the
+`pod_patch` pipeline parameter.
+The `pod_patch` parameter takes a string that specifies the changes
+that you want to add to your existing manifest. To create
+a patch, you need to generate a diff of the original ReplicationController
+and the one with your changes. You can use one of the online JSON patch
+utilities, such as [JSON Patch Generator](https://extendsclass.com/json-patch.html)
+to create a diff. A diff for mounting a volume might look like this:
+
+```json
+[
+ {
+  "op": "add",
+  "path": "/volumes/-",
+  "value": {
+   "name": "task-pv-storage",
+   "persistentVolumeClaim": {
+    "claimName": "task-pv-claim"
+   }
+  }
+ },
+ {
+  "op": "add",
+  "path": "/containers/0/volumeMounts/-",
+  "value": {
+   "mountPath": "/data",
+   "name": "task-pv-volume"
+  }
+ }
+]
+```
+
+This output needs to be converted into a one-liner and added to the
+pipeline spec.
+
+We will use the [OpenCV example](../getting_started/beginner_tutorial/).
+to demonstrate this functionality.
+
+To mount a volume, complete the following steps:
+
+1. Create a PersistentVolume and a PersistentVolumeClaim as
+described in [Configure a Pod to Use a PersistentVolume for Storage](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/). Modify `mountPath` and `path` as needed.
+
+   For testing purposes, you might want to add an `index.html`
+   file as described in [Create an index.html file](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-an-index-html-file-on-your-node).
+
+1. Get the ReplicationController (RC) manifest from your pipeline:
+
+   ```bash
+   kubectl get rc <rc-pipeline> -o json > <filename>.yaml
+   ```
+
+   **Example:**
+
+   ```bash
+   kubectl get rc pipeline-edges-v7 -o json > test-rc.yaml
+   ```
+
+1. Open the generated RC manifest for editing.
+1. Under `spec`, find the `volumeMounts` section.
+1. Add your volume in the list of mounts. 
+
+   **Example:**
+
+   ```json
+   {
+        "mountPath": "/data",
+        "name": "task-pv-storage"
+   }
+   ```
+
+   `mountPath` is where your volume will be mounted inside of the
+   container.
+
+1. Find the `volumes` section:
+1. Add the information about the volume.
+
+   **Example:**
+
+   ```bash
+   {
+        "name": "task-pv-storage",
+        "persistentVolumeClaim": {
+            "claimName": "task-pv-claim"
+        }
+   }
+   ```
+
+   In this section, you need to specify the PersistentVolumeClaim you have
+   created in Step 1.
+
+1. Save these changes to a new file.
+1. Copy the contents of the original RC to the clipboard:
+
+   * If you are on macOS, you can use `pbcopy`. Example:
+
+     ```bash
+     pbcopy < test.yaml
+     ```
+
+   * If you are on Linux, you can use [xclip](https://linux.die.net/man/1/xclip).
+     Example:
+
+     ```bash
+     xclip -sel clip < test.yaml
+     ```
```

Comment:
This is X11-specific, so I'm not sure if it'd work on, e.g., wayland. Honestly I'd just leave out the command to copy to clipboard, at least for linux; users should know how to do that.

Reply:
removed